source/jormungandr/jormungandr/realtime_schedule/timeo.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, id, service_url, service_args, timezone, timeout=10):
         self.service_url = service_url
         self.service_args = service_args
         self.timeout = timeout  # timeout in seconds
-        self.id = id
+        self.service_id = id
```

Comment:
I don't like service, there is a lot of this already called "service" in the transport universe